CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## 2.5.0 (2023-06-29)
+
+## What's Changed
```

Comment:
Just a nitpick, but this title should be level 3 and not level 2.
Is it possible to fix that easily (no problem is it is not possible)?

Reply:
agreed, but i don't know how to customize it :(

Reply:
No problem, it is not that important :)

Reply:
https://github.com/googleapis/release-please has quite a bit of configurability and all kinds manifests and configs so maybe there's a way. i just configured it to use the github release API to get those nice entries with PR links and authors :)

Reply:
I just did a quick search, but I didn't find anything related to the markdown format.